astroid/interpreter/objectmodel.py

```diff
@@ -493,8 +493,8 @@ def __init__(self):
         super().__init__()
 
     @property
-    def attr___annotations__(self) -> node_classes.Unkown:
-        return node_classes.Unknown()
+    def attr___annotations__(self) -> node_classes.Unknown:
```

Comment:
Nice catch, strange that this typo was not a problem anywhere.

Reply:
My local checker (not mypy, pyright) found other typing issues in that file. Maybe mypy isn't running on this file for whatever reason.

Reply:
Yeah, mypy doesn't run on a lot of file at the moment. Lots of dynamic things that are very hard to type. I would have thought that actual import of something that does not exist would have been an error, but it seems it's not.

Reply:
I think it's due to `from __future__ import annotations` at the top. It makes python treat every annotation as a string. Without it, it would have been checked.